include/private/fnft__misc.h

```diff
@@ -235,6 +236,66 @@ FNFT_UINT fnft__misc_nextpowerof2(const FNFT_UINT number);
 FNFT_INT fnft__misc_resample(const FNFT_UINT D, const FNFT_REAL eps_t, FNFT_COMPLEX const * const q,
     const FNFT_REAL delta, FNFT_COMPLEX *const q_new);
 
+/**
+ * @brief Multiples two square matrices of size N.
+ *
+ * @ingroup  misc
+ *
+ * Multiples two square matrices U and T of size N. T is replaced by the
+ * result U*T.
+ * @param[in] N Positive integer that is the size of the two matrices.
+ * @param[in] U Pointer to the first element of complex values NxN matrix U.
+ * @param[in,out] T Pointer to the first element of complex values NxN matrix T.
+ * Contains the result U*T on return.
+ */
+static inline void fnft__misc_square_matrix_mult(const FNFT_UINT N, FNFT_COMPLEX * const U,
```

Comment:
This is minor, but `mat_mult_NxN` would be consistent with how the `poly_fmult` functions are named.

Reply:
So should I move them to a new header `fnft__mat_mult.h`?

Reply:
I think it is fine if they stay in misc. These are simple helper functions.

Reply:
Okay will only change the names then.

Reply:
Should these be `fnft__misc_mat_mult_NxN` or just `fnft__mat_mult_NxN` ?

Reply:
`fnft__misc_mat_mult_NxN`